cmd/spec/path.yaml

```diff
@@ -28,6 +28,8 @@ paths:
   /quickstarts/{id}:
     get:
       summary: Return a quickstarts by ID
+      parameters:
+      - $ref: '#/components/schemas/queryParams/Id'
```

Comment:
You have to do something similar for the `/helptopics/{name}:` endpoint.

Reply:
Fixed